microsoft-365/admin/manage/manage-feedback-ms-org.md

```diff
@@ -75,12 +75,17 @@ Your devices must be on a minimum build number to use these policies. See the ta
 
 |**Policy name**|**Default state**|**Control summary**|
 |:-----|:-----|:-----|
+|Allow users to access feedback portal|On|Manage user access to the Feedback portal|
 |Allow users to submit feedback to Microsoft|On|Controls feedback entry points across applications|
 |Allow users to receive and respond to in-product surveys from Microsoft|On|Controls survey prompts within product|
 |Allow users to include screenshots and attachments when they submit feedback to Microsoft|Off|Determines what metadata the user can decide to submit with feedback/survey|
 |Allow Microsoft to follow up on feedback submitted by users|Off|Determines if user can share contact info with feedback/survey|
 |Allow users to include log files and content samples when feedback is submitted to Microsoft|Off|Determines metadata the user can decide to submit with feedback/survey|
 
+> [!NOTE]
+> The ***Allow users to access the Feedback portal*** policy is a cloud policy. This policy is not defined in ADMX and does not have a corresponding registry key available to set this policy. Customers must create a cloud policy to enforce. The reason this is a cloud policy is because the feedback portal is a web application that makes a call to the cloud policy service which is also a web application requesting the policies for the user that is signing in. If this policy is configured, the feedback portal will receive the configured policy value in the response from the cloud policy service.
```

Comment:
```suggestion
> The **Allow users to access the feedback portal** policy is a cloud policy. This policy is not defined in ADMX and does not have a corresponding registry key available to set the policy. You should create a cloud policy to enforce it. This is a cloud policy because the feedback portal is a web application that makes a call to the cloud policy service, which is also a web application, requesting the policies for the user who signs in. If this policy is configured, the feedback portal will receive the configured policy value in the response from the cloud policy service.
```